server/graphql/invoice_line/src/mutations/inbound_shipment_line/line/insert.rs

```diff
@@ -88,6 +89,7 @@ impl InsertInput {
             total_before_tax,
             tax_percentage,
             item_variant_id,
+            vvm_status_id: _, // Ignoring until the ability to record vvm status is added by https://github.com/msupply-foundation/open-msupply/issues/7366
```

Comment:
Thank you for comment, that is helpful

Reply:
Can you add this as a comment in the issue as well so it doesn't get missed?